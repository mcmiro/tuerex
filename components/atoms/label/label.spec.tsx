/* eslint-disable no-use-before-define */
import * as React from 'react';
import { screen, render } from '@testing-library/react';
import Label from './';

describe('Test label component', () => {
  it('Should pass when content is additional', () => {
    render(<Label content="test" data-testid="label-test" />);

    const label = screen.queryByTestId('testid');
    expect(label).toBeDefined();
    expect(label).toMatchSnapshot();
  });
});
