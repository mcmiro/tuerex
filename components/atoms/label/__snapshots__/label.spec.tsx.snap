// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test label component Should pass when content is additional 1`] = `null`;
